webapp/src/main/java/org/opentestsystem/rdw/admin/Application.java

```diff
@@ -24,7 +25,8 @@
         EndpointAutoConfiguration.class,
         StatusConfiguration.class,
         YamlPropertiesConfigurator.class,
-        DataSourceConfiguration.class
+        DataSourceConfiguration.class,
+        JdbcOrganizationRepository.class
```

Comment:
This is a @Component in RDW_Common. This is the least impactful way to force it into this project; component scanning caused some odd behavior with the controllers.